### DISCUSSION TITLE
Does ipcam-combined accuracy depend on Size and Angular orientation of objects within image

### DISCUSSION THREAD

**Discussion by GarageKing** [codeproject/CodeProject.AI-Server#243] [Q&A]
I'm currently using ipcam-combined with Blue Iris on low powered PC and Yolov5.net.  System easily recognizes vehicles, people and a few large dogs.  Seems to have difficulty detecting smaller dogs farther away in the images.  Happens with both low and high contrast images. Camera is rotated about 30 degrees.
Does camera/image rotation affect accuracy?
Do any parameters affect minimum object size?
![Person and dog](https://github.com/user-attachments/assets/8a2a0e71-dddf-404b-8ecc-860d47755c02)

**Comment by GarageKing** [comment]
The model was able to detect the dog when I cropped the image.  This indicates that the size or proportion of pixels pixels that an object uses within an image is a factor.  Is there a way to have the model recognize the dog as a smaller object in the larger image.
![Full](https://github.com/user-attachments/assets/d28372f1-9f61-41a9-a1fb-be590beb92d1)

![Cropped](https://github.com/user-attachments/assets/8255bccf-c69f-4644-bab9-758dbbb48f74)

**Comment by tbc725** [comment]
Does camera/image rotation affect accuracy?

Yes it does to a degree. See below example (upside down is a bit dramatic, but at least it gives the idea of that it does affect accuracy to a degree).
![1](https://github.com/user-attachments/assets/3051be6a-59e6-471b-8821-f006d3719c85)
![2](https://github.com/user-attachments/assets/b8051259-b93c-4dbe-9c2e-33df53659d96)

Is there a way to have the model recognize the dog as a smaller object in the larger image.
Not guaranteed, but you can try to use the default model instead of the ipcam-combined, and then change the model to large model from the dashboard. (See below)

![3](https://github.com/user-attachments/assets/2d4ef404-e008-4a0d-a914-9224ddb53956)

Since the model is bigger, it should be trained with more data, and hopefully it will be able to recognize the dog in your image.

If that doesn't work, you could try YOLOv8 and try its large or huge model, or the segmentation mode to see if it catches it.

![4](https://github.com/user-attachments/assets/b34abefe-6828-44d7-99b1-7f588aa5176e)
![5](https://github.com/user-attachments/assets/8131f2d0-86a4-43fa-8509-1fbd0b613ff7)
![6](https://github.com/user-attachments/assets/c72c3159-b280-4c09-bf65-fcc2869d925b)

Hope this help.
